btree/btree.c

```diff
@@ -4279,6 +4286,7 @@ void m0_btree_del_credit(const struct m0_btree  *tree,
 	node_free_credit(tree->t_desc->t_root, ksize, vsize, &cred);
 	m0_be_tx_credit_mul(&cred, MAX_TREE_HEIGHT);
 
+	btree_callback_credit(&cred);
```

Comment:
if we call btree_callback_credit here, for nr > 1,for accum,  tc_cb_nr will be still equal to BTREE_CALLBACK_CREDIT where as it should be nr * BTREE_CALLBACK_CREDIT as m0_be_tx_credit_mul is not modifying tc_cb_nr. That is every time tc_cb_nr will b equal to BTREE_CALLBACK_CREDIT irrespective of nr.